includes/class-llms-order-generator.php

```diff
@@ -0,0 +1,733 @@
+<?php
+/**
+ * LLMS_Order_Generator class file.
```

Comment:
```suggestion
 * LLMS_Order_Generator class file
```
https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#file-headers-1
https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#6-file-headers